### ISSUE TITLE
Style guide document to wiki

### ISSUE THREAD

**Issue by hbh7** [PollBuddy/PollBuddy#314]
**Please describe what has to be done**
Create a wiki page for details from the style guide document. It can all go on one page I think. Also make a link to the gdrive branding folder in the Resources repo for logos/font.

**Additional context**
https://docs.google.com/document/d/1V8RTIty-MY-kqMPtiyi_PEgDJ4GlCvOBaiaiKVWxXPw/edit

**Comment by aleong1** [comment]
How would I make a pull request/review for this because I didn't make a new branch

**Comment by hbh7** [comment]
Probably just post a link here I'd say since the wiki doesn't really work with branching.

**Comment by aleong1** [comment]
ok I can just write what I did here so it's like a pull request

Link to style guide: https://github.com/PollBuddy/PollBuddy/wiki/Style-Guide

I just moved everything that was in the Google Drive style guide to this Wiki page. I also put a link to the google drive branding folder that's in the resources repo. Also some colors are that are listed in the style guide (Light Gray and Dark Gray / Near Black) aren't used in any of the css files so I left it blank for what it was used for but I still put in in the table in case someone uses it in the future.
